fobi_custom/plugins/form_handlers/collect_data/fobi_form_handlers.py

```diff
@@ -54,7 +54,11 @@ def run(self, form_entry, request, form, form_element_entries=None):
             plugin_data = element.plugin_data
             json_plugin_data = json.loads(plugin_data)
 
-            try:
+            # Check if an element is help text. If it is, skip it
+            if 'text' in json_plugin_data.keys():
+                pass
```

Comment:
I think a `continue` would be more explicit here, to indicate that we want to skip this form element entirely. Any reason you prefer `pass`?

Reply:
Nope continue makes more sense!